x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.test.tsx

```diff
@@ -368,11 +380,153 @@ describe('actions_connectors_list', () => {
     });
   });
 
+  describe('check EditConnectorFlyout will open on edit connector', () => {
```

Comment:
It makes sense to have a test here to make sure that `expect(editItem).toBeCalled();` but if the edit connector logic was moved to `actions_connectors_home.tsx`, I think we also need some `EditConnectorFlyout` tests in `actions_connectors_home.test.tsx`.

At least something like `it('shows "Edit connector" flyout when a connector name is clicked', ...`.

Reply:
The problem here, that in `actions_connectors_home` we do not have connectors to edit, just header.  But we have it covered by functional test.